Frameworks/UIKit/WYPopoverController.m

```diff
@@ -1459,10 +1463,15 @@ - (CGRect)innerRect:(CGRect)rect arrowDirection:(WYPopoverArrowDirection)aArrowD
         result.size.height -= borderWidth;
     }
     
+    // The below code is disabled on WINOBJC due to some issue where |viewContentInsets| has a garbage value.
+    // Given that most of the theming/drawing code in WYPopoverController is effectively a no-op in WINOBJC,
+    // just skipping over the below is an adequate (or at least quick) fix.
```

Comment:
Do you have a standalone repro that we can look at to locate the underlying issue?